packages/shorebird_cli/lib/src/android_sdk.dart

```diff
@@ -116,7 +116,7 @@ class AndroidSdk {
   }
 
   /// The path to the `adb` executable.
-  late final adbPath = getPlatformToolsPath(
+  late final String? adbPath = getPlatformToolsPath(
```

Comment:
I don't think we should make all these type inference changes unless there's a clear reason. The analyzer can infer the types just fine so I don't think there's much value in arbitrarily specifying types in some places.